nil pointer dereference

## What happened?

Since some getting started is missing I simply used docker  and a example md file which causes an exception. Root cause may be incorrect usage, but I have no clue what is missing.

`docker run -v $(pwd):/docs --rm -i kovetskiy/mark:latest mark -u <myuser> -p <mypass> -l <myurl>....sandbox?pageId=1234" -f list.md --dry-run`

it gives
```2025-05-15 09:30:29.622 INFO  processing list.md
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x20 pc=0x9e7725]

goroutine 1 [running]:
github.com/kovetskiy/mark/util.processFile({0xc000219d39, 0x7}, 0xc000215b60, 0xc000025780, {0x7ffccdeeef46?, 0xa}, {0x7ffccdeeeef0, 0x8}, 0xc00029fab7)
	/go/src/github.com/kovetskiy/mark/util/cli.go:137 +0x2c5
github.com/kovetskiy/mark/util.RunMark(0xc000025780)
	/go/src/github.com/kovetskiy/mark/util/cli.go:85 +0x6f7
github.com/urfave/cli/v2.(*Command).Run(0xc0001ee2c0, 0xc000025780, {0xc000022140, 0xa, 0xa})
	/go/pkg/mod/github.com/urfave/cli/v2@v2.27.6/command.go:276 +0x7e2
github.com/urfave/cli/v2.(*App).RunContext(0xc0001c4600, {0xeacce0, 0x12ae8e0}, {0xc000022140, 0xa, 0xa})
	/go/pkg/mod/github.com/urfave/cli/v2@v2.27.6/app.go:333 +0x5a5
github.com/urfave/cli/v2.(*App).Run(...)
	/go/pkg/mod/github.com/urfave/cli/v2@v2.27.6/app.go:307
main.main()
	github.com/kovetskiy/mark/main.go:44 +0x186

```


## What did you expect to happen?

Some thing is rendered to html

## How can we reproduce the behavior you experienced?

see above. list.md is from examples directory


In case this is related to specific markdown, please provide a minimal markdown example here.

## Further Information (please complete the following information)

* Mark Version (`mark --version`): 12.2.0
* Mark Parameters: see above
* Confluence Hosting: Datacenter
* Confluence Version: 8.5.21
* Environment specific Information: docker

## Logs or other output

Please provide logs, other kind of output here.

## Additional context

Add any other context about the problem here.

--- a/util/cli.go
+++ b/util/cli.go
@@ -134,14 +134,16 @@ func processFile(
 		return nil
 	}
 
-	if meta.Space == "" {
-		fatalErrorHandler.Handle(nil, "space is not set ('Space' header is not set and '--space' option is not set)")
-		return nil
-	}
+	if meta != nil {
+		if meta.Space == "" {
+			fatalErrorHandler.Handle(nil, "space is not set ('Space' header is not set and '--space' option is not set)")
+			return nil
+		}
 
-	if meta.Title == "" {
-		fatalErrorHandler.Handle(nil, "page title is not set ('Title' header is not set and '--title-from-h1' option and 'h1_title' config is not set or there is no H1 in the file)")
-		return nil
+		if meta.Title == "" {
+			fatalErrorHandler.Handle(nil, "page title is not set ('Title' header is not set and '--title-from-h1' option and 'h1_title' config is not set or there is no H1 in the file)")
+			return nil
+		}
 	}
 
 	stdlib, err := stdlib.New(api)

